interfaces/builtin/network_status.go

```diff
@@ -24,131 +24,82 @@ import (
 
 	"github.com/snapcore/snapd/interfaces"
 	"github.com/snapcore/snapd/interfaces/apparmor"
-	"github.com/snapcore/snapd/interfaces/dbus"
 	"github.com/snapcore/snapd/snap"
 )
 
-const networkStatusSummary = `allows operating as the NetworkingStatus service`
+const networkStatusSummary = `allows access to network connectivity status`
 
 const networkStatusBaseDeclarationSlots = `
   network-status:
     allow-installation:
       slot-snap-type:
         - app
+        - core
     deny-connection: true
 `
 
 const networkStatusPermanentSlotAppArmor = `
-# Description: Allow owning the NetworkingStatus bus name on the system bus
```

Comment:
networkStatusPermanentSlotAppArmor can go away if we remove `- app` as a slot-snap-type.